fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -63,35 +68,64 @@ def __init__(self, fapd_manager):
                     sensitivity_func=self.analyze_button_sensitivity,
                 )
             ],
+            "clear": [
+                UIAction(
+                    "Clear",
+                    "Clear Fields",
+                    "edit-clear",
+                    {"clicked": self.on_clearButton_clicked},
+                )
+            ],
         }
 
         UIPage.__init__(self, actions)
         self._fapd_profiler = FaProfiler(fapd_manager)
         self.running = False
+        self.inputDict = {}
         self.analysis_available = False
 
     def analyze_button_sensitivity(self):
         return self.analysis_available
 
+    def on_next_system(self, system):
+        if not self.inputDict == system.get("profiler").entry:
+            self.update_field_text(system.get("profiler").entry)
+
+        buff = self.get_object("profilerOutput").get_buffer()
+        text = buff.get_text(buff.get_start_iter(), buff. get_end_iter(), True)
+        if not text == system.get("profiler").output:
+            self.update_output_text(system.get("profiler").output)
```

Comment:
If you did something like
```
self.analysis_available=bool(system.get("profiler").output)
```
here would it enable the Analyzer button when there was existing output? That way if the user came back here to look at some output they could go back to the Analyzer using the toolbar button.

Reply:
This doesn't seem to be working. Do we need a toolbar refresh to get the button to become active? I
1. run profiler
2. clicked to go to Analyzer
3. went back to the profiler
4. the entries and output were reloaded, but the Analyzer button was not active until I ran again

Reply:
Also the persistent checkbox does not seem to get repopulated.

Reply:
I'm seeing the markup show as plaintext instead of the bolding

Reply:
> I'm seeing the markup show as plaintext instead of the bolding

That should be fixed with my last commit.